app/components/message_draft_component.html.erb

```diff
@@ -6,26 +6,24 @@
 
       <%= render MessageDraftAttachmentsListComponent.new(message: @message) %>
 
-      <% if @message.not_yet_submitted? %>
         <div class="flex justify-between w-full">
-          <div class="gap-2 p-6 border-t-0 border-r-0 border-b border-l-0 border-gray-200">
+          <% if @message.correctly_created? %>
+            <div class="gap-2 p-6 border-t-0 border-r-0 border-b border-l-0 border-gray-200">
             <div id="<%= dom_id(@message, :submission) %>" is="turbo-frame" class="flex gap-2">
-              <% if @message.not_yet_submitted? %>
-                <%= form_with(url: submit_message_draft_path(@message), method: :post, data: { turbo_frame: "_top" }, local: true) do |form| %>
-                  <%= form.submit "Odoslať", class: "flex justify-strech items-start self-stretch flex-grow-0 flex-shrink-0 gap-2 p-6 text-white font-medium rounded-lg text-sm px-3.5 py-2.5 bg-blue-700 hover:bg-blue-800 dark:bg-blue-600 dark:hover:bg-blue-700 focus:ring-4 focus:ring-blue-300 focus:outline-none dark:focus:ring-blue-800" %>
-                <% end %>
+              <%= form_with(url: submit_message_draft_path(@message), method: :post, data: { turbo_frame: "_top" }, local: true) do |form| %>
+                <%= form.submit "Odoslať", class: "flex justify-strech items-start self-stretch flex-grow-0 flex-shrink-0 gap-2 p-6 text-white font-medium rounded-lg text-sm px-3.5 py-2.5 bg-blue-700 hover:bg-blue-800 dark:bg-blue-600 dark:hover:bg-blue-700 focus:ring-4 focus:ring-blue-300 focus:outline-none dark:focus:ring-blue-800" %>
```

Comment:
Toto vyzera komplikovane, ale jedna sa iba o upravu podmienok. Neboli spravne nastavene. Aktualne:
- odoslat sa da iba validny draft
- zmazat sa da draft bez ohladu na to, ci je validny